proto/options.proto

```diff
@@ -227,4 +227,5 @@ message Options {
   optional bool linear_references = 45;                                   // Include linear references for graph edges returned in certain responses.
   repeated CostingOptions recostings = 46;                                // Costing options to use to recost a path after it has been found
   repeated Ring exclude_polygons = 47;                                    // Rings/polygons to exclude entire areas during path finding
+  optional bool prioritize_bidirectional = 48 [default = false];          // Prioritize bidirectional a* over timedependent_forward a*.
```

Comment:
yep, but only when a time dependent (depart_at arrive_by or current time) route is requested!